app/helpers/teacher_sets_es_helper.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module TeacherSetsEsHelper
+  # Make teacherset object from elastic search json
+  def create_ts_object_from_es_json(json)
+    arr = []
```

Comment:
Maybe give this variable a more descriptive name — like `teacher_sets`  or `teacher_sets_from_es`